oak-doc/src/site/markdown/query/lucene.md

```diff
@@ -408,6 +408,14 @@ nodeScopeIndex
   explicit property definition provided then it would only be included if
   `nodeScopeIndex` is set to true.
 
+  Note : If an index definition consists of any property with nodeScopeIndex set to
+  true, then it will index the node name property for all the nodes (with nodeType
+  matching to or a child type of the one defined in the indexRule).
+  This could result in large index size in case of indexRules on broader nodeTypes such as nt:base.
```

Comment:
Nit: I would use "node type" instead "nodeType" in this case, as here we refer to the "concept" and not the term "nodeType". But it's just my preference... It's not wrong of course to use "nodeType" and it's clear also.

Reply:
sure..I changed it to node type